### DISCUSSION TITLE
GDALGridOptions

### DISCUSSION THREAD

**Discussion by eddyable** [MaxRev-Dev/gdal.netcore#155] [Q&A] [answered]
Cannot instantiate the GDALGridOptions
```
// Define the grid options
string[] options = new[] {
    "algorithm=invdist:power=2.0:smoothing=1.0",
    "output_type=Float32"
};

// Create the GDALGridOptions
var gridOptions = new GDALGridOptions(options: options);
```

it throws an exception: Zero positional arguments expected.

**Comment by MaxRev-Dev** [comment, marked as answer]
https://gdal.org/en/latest/programs/gdal_grid.html#gdal-grid

The options you have specified are wrong.
It should go like this:
`"-of", "gtiff","-ot", "Float64", “-a”, “…”`

**Reply by eddyable** [reply]
Thank you! 💪
